std-ddox.ddoc

```diff
@@ -3,6 +3,14 @@ XREF = std.$1.$2
 XREF_PACK = std.$1.$2.$3
 CXREF = core.$1.$2
 ECXREF = etc.$1.$2
+MREF_HELPER=/$1$(MREF_HELPER $+)
+MREF_HELPER2=.$1$(MREF_HELPER2 $+)
+MREF=<a href="/library/$1$(MREF_HELPER $+).html">$1$(MREF_HELPER2 $+)</a>
 LREF = $1
 ROOT_DIR=/
+NOTE = <div class="note">$0</div>
```

Comment:
I don't think you need to redefine things in std-ddox.ddoc if the values are the same as in std.ddoc, AFAIU this file is essentially an override of std.ddoc for DDox.